src/Coriolis/constant_cartesian_coriolis.jl

```diff
@@ -29,7 +29,7 @@ constant rotation can be specified in three different ways:
   (which defaults to Earth's rotation rate).
 """
 function ConstantCartesianCoriolis(FT=Float64; fx=nothing, fy=nothing, fz=nothing,
-                                               f=nothing, rotation_axis=ZDirection(), 
+                                               f=nothing, rotation_axis=NegativeZDirection(),
```

Comment:
FYI, I realized this is wrong. `ZDirection` should still exist and be used in `ConstantCartesianCoriolis`. So let's not merge this as is.

I'm working on a second attempt at this PR making sure that doctests are passing at every step. That way I can pinpoint what's causing the error here.

Reply:
could be nice also to define 

```julia
Base.-(::ZDirection) = NegativeZDirection()
Base.-(::NegativeZDirection) = ZDirection
````